home.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>First Moblie Project </title>
+  <script src="https://kit.fontawesome.com/d36fe61649.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?famliy=Poppins:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <header>
+    <nav class="nav_bar">
+        <div class="container">
+           <div class="navbar_logo"><a href="https://github.com/GrebilCruz">Grebil Cruz</a></div>
+           <div class="navbar_icon"><a href="#"><img alt="burger" src="image/burger.png"> </a></div>
+        </div>
+    </nav>
+  </header>
+  <section id="headline">
+    <div class="greeting">
+      <h1>I'm Grebil, Glad to See you!</h1>
+    </div>
+    <p>I'm a software developer! I can help you build a product, feature or website Look through some of my work and
+      experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <p class="connect">
+      Let's Connect!
+    </p>
```

Comment:
- [ ] Please remove the `p tag` here and use any of the `h2, h3, h4, h5, h6 tag`.